app/src/main/java/com/example/minerva/di/RepositoryModule.kt

```diff
@@ -21,10 +20,9 @@ object RepositoryModule {
     fun provideMainRepository(
```

Comment:
I added a comment about how to handle this in the other pr please take a look.

Reply:
i did i will request another pr